src/renderer/components/cluster-settings/components/cluster-node-shell-setting.tsx

```diff
@@ -95,15 +78,25 @@ export class ClusterNodeShellSetting extends React.Component<Props> {
         <section>
           <SubTitle title="Image pull secret" id="image-pull-secret"/>
           <Input
-            placeholder={"Add a secret name..."}
+            placeholder="Specify a secret name..."
             theme="round-black"
             value={this.imagePullSecret}
-            onChange={this.onSecretChange}
-            onBlur={this.saveSecret}
-            iconRight={<Icon small material="close" onClick={this.clearSecret} tooltip="Clear"/>}
+            onChange={value => this.imagePullSecret = value}
+            iconRight={
+              this.imagePullSecret
+                ? (
+                  <Icon
+                    smallest
+                    material="close"
+                    onClick={() => this.imagePullSecret = ""}
+                    tooltip="Clear"
+                  />
+                )
+                : undefined
+            }
           />
           <small className="hint">
-            Name of a pre-existing secret (optional). Used for pulling image from a private registry.
+            Name of a pre-existing secret. An optional setting. Used for pulling image from a private registry.
```

Comment:
Will it be a good idea that this message also specifies the "Namespace" for the pre-existing secret i.e. kube-system ??